Cargo.toml

```diff
@@ -35,9 +35,12 @@ transform-types = []
 
 # libm is required when building no_std
 libm = ["num-traits/libm"]
+#
+crevice_traits = ["crevice", "bytemuck"]
 
 [dependencies]
 bytemuck = { version = "1.5", optional = true, default-features = false }
+crevice = { version = "0.6", optional = true, git = "https://github.com/mtsr/crevice.git", branch = "develop" }
```

Comment:
I wouldn't want to merge until this can use crates.io, I assume they need to make a new release or something?